modules/KalturaSupport/resources/mw.KWidgetSupport.js

```diff
@@ -113,6 +114,7 @@ mw.KWidgetSupport.prototype = {
 		
 		// Add hook for check player sources to use local kEntry ID source check:
 		embedPlayer.bindHelper( 'checkPlayerSourcesEvent', function( event, callback ) {
+			_this.originalStreamerType = embedPlayer.getKalturaConfig( null, 'streamerType' ) ? embedPlayer.getKalturaConfig( null, 'streamerType' ) : 'auto';
```

Comment:
@amirch1 if we dont have streamer type why do we add auto?
the streamer type will tell the player to use kplayer and not html5 when possible